docs/manuals/userman/gui/viselements/generic/metric.md_template

```diff
@@ -1,14 +1,272 @@
+or a circular gauge.<br/>
+The control can also display a *delta* value, which is meant to give a hint about how the metric
+value is evolving.
+
 # Details
 
-TODO
+The `metric` control provides a graphical representation of a value in a quantitative context.<br/>
+This value can result from a significant computational step or a KPI (Key Performance Indicator) in
+an industrial context.<br/>
+In Taipy GUI, this control is implemented using
+[Plotly's gauge charts](https://plotly.com/javascript/gauge-charts/).
+
+The purpose of the control is to help evaluate a numerical value by graphically representing the
+value in a context using a gauge.<br/>
+A quick look at the control immediately gives relevant information, yet the control still shows
+precise figures that a user can investigate further.
+
+In addition to the main value, the `metric` control also lets you complement the graphs with a
+*delta* value that is meant to express the variation of the main value.
+
+The control provides ways to customize the underlying gauge by setting its global range or the
+colors for sections of that range.
 
 # Styling
 
-All the date controls are generated with the "taipy-metric" CSS class. You can use this class
-name to select the date selectors on your page and apply style.
+All the metric controls are generated with the "taipy-metric" CSS class. You can use this class
+name to select the metric controls on your page and apply style.
 
 # Usage
 
-TODO
+## Simple case {data-source="gui:doc/examples/controls/metric-simple.py"}
+
+The most common use case for the `metric` control is displaying a value on a gauge, with a *delta*
+value indicating how much the value has increased or decreased compared to a previous state.<br/>
+The [*value*](#p-value) property (the default property for this control) holds the value to be
+represented. You can set the [*delta*](#p-delta) property to the value variation.<br/>
+You can also add an indicator of a specific value, such as a threshold value, by setting the
+[*threshold*](#p-threshold) property to the value of that parameter.
+
+Say you have declared the following variables:
+```python
+value = 72
+delta = 15
+threshold = 60
+```
+
+The following definition of the control:
+!!! taipy-element
+    default={value}
+    delta={delta}
+    threshold={threshold}
+
+Renders as such:
+<figure class="tp-center">
+    <img src="../metric-simple-d.png" class="visible-dark"  width="80%"/>
+    <img src="../metric-simple-l.png" class="visible-light" width="80%"/>
+    <figcaption>Simple case</figcaption>
+</figure>
+
+## Setting the gauge type {data-source="gui:doc/examples/controls/metric-type.py"}
+
+You can change the representation to an horizontal bar by setting the [*type*](#p-type) 
+
+Here is the definition of a `metric` control that is horizontal:
+!!! taipy-element
+    default={value}
+    delta={delta}
+    threshold={threshold}
```

Comment:
where is the type parameter ?

Reply:
That's right. Something went very wrong here.
On it. Thanks.